client/src/components/TypeList.vue

```diff
@@ -74,45 +75,47 @@ export default Vue.extend({
     <v-subheader class="flex-shrink-0">
       Type Filter
     </v-subheader>
-    <div class="overflow-y-auto flex-grow-1">
-      <v-container>
+    <div class="overflow-y-auto">
+      <v-container class="py-2">
         <v-row
           v-for="type in allTypes.value"
           :key="type"
           class="hover-show-parent"
         >
-          <v-checkbox
-            v-model="checkedTypes.value"
-            :value="type"
-            :color="typeStyling.value.color(type)"
-            :label="type"
-            dense
-            shrink
-            hide-details
-            class="my-1 ml-3 type-checkbox"
-          />
-          <v-spacer />
-          <v-tooltip
-            open-delay="100"
-            bottom
-          >
-            <template #activator="{ on }">
-              <v-btn
-                class="mr-1 hover-show-child"
-                icon
-                small
-                v-on="on"
-                @click="clickEdit(type)"
-              >
-                <v-icon
+          <v-col class="d-flex flex-row align-center py-0">
+            <v-checkbox
+              v-model="checkedTypes.value"
+              :value="type"
+              :color="typeStyling.value.color(type)"
+              :label="type"
+              dense
+              shrink
+              hide-details
+              class="my-1"
+            />
+            <v-spacer />
+            <v-tooltip
+              open-delay="100"
+              bottom
+            >
```

Comment:
I didn't like the behavior for long type names:
![image](https://user-images.githubusercontent.com/61746913/86413342-f3ce8f80-bc8e-11ea-9076-a11f43f5f6ce.png)

Much rather find a way to do wrapping like this with the editing button always in the same position and wrapping of the name below the current below:
![image](https://user-images.githubusercontent.com/61746913/86413367-09dc5000-bc8f-11ea-8883-a1e465673863.png)

Reply:
Reverted the removal of `type-checkbox`, this appears to be working now.